plugins/include/open62541/server_config_default.h

```diff
@@ -243,6 +239,111 @@ UA_ServerConfig_addAllSecureSecurityPolicies(UA_ServerConfig *config,
                                        const UA_ByteString *certificate,
                                        const UA_ByteString *privateKey);
 
+#ifdef __linux__ /* Linux only so far */
+
+/* Adds the security policy ``SecurityPolicy#Basic128Rsa15`` to the server. A
+ * server certificate may be supplied but is optional.
```

Comment:
Well, that's really cool! The server certificate being optional on security policy construction time is definitely a feature!
Nevertheless, the private key seems to be still required here, is that right? At least a look into the "UA_mbedTLS_LoadPrivateKey" function does not state otherwise and it should result in an error if an empty private key is provided.

Reply:
That was a copying mistake. A certificate and a private key are required to create a security policy.